src/test/java/tests/ApiQuotaManagerTest.java

```diff
@@ -0,0 +1,104 @@
+package tests;
+
+import org.junit.Test;
+import util.ApiQuotaManager;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests the correct behaviour of the methods in the ApiQuotaManager class.
+ */
+public class ApiQuotaManagerTest {
+
+    @Test
+    public void computeRefreshTimerPeriod_zeroApiCallsUsedInRecentRefresh() {
+
+        long result = ApiQuotaManager.computeRefreshTimerPeriod(100, 10, 0, 50, 1);
+        assertEquals(result, 1);
```

Comment:
As the test is simple, I don't mind if you merge all of them into one `@Test`. To improve readability, you can extract a utility method and write tests like this:
```java
verify(100, //remainingApiQuota
       10,  //minutesToNextQuotaTopup
       0,   //apiCallsUsedInRecentRefresh
       50,  //apiQuotaBuffer
       1,   //minRefreshPeriod
       1);  //explanation of why this is the expected answer
```